lib/launcher.ts

```diff
@@ -112,7 +112,7 @@ let initFn = function(configFile: string, additionalConfig: Config) {
       .then(() => {
 
         return q
-            .Promise<any>((resolve: Function) => {
+            .Promise<any>((resolve: Function, reject: Function) => {
```

Comment:
This is not possible.  Setting this to void returns a transpile error. `lib/launcher.ts(114,16): error TS2350: Only a void function can be called with the 'new' keyword.`